### PR TITLE
Show Cody walkthrough help text

### PR DESCRIPTION
**Pull request by umpox**
## Description

Updates the Cody welcome text for VS Code so that it now links to the walkthrough instead of the docs site.

image

> **Note**
> When working on this I discovered an issue that means the `openWalkthrough` command doesn't work reliably - especially when an editor has just been opened. I have found a work around for this that works well for now. We would also have this issue with the menu link to "Getting Started" so I think we should continue and merge this.
> I have opened an issue in VS Code for this [here](https://github.com/microsoft/vscode/issues/186165).

## Test plan

Tested locally.
1. Open Cody window in VS Code
2. Click "Getting Started" link

### CODE REVIEW

File: client/cody-shared/src/chat/markdown.ts [hunk 1]
```diff
@@ -2,6 +2,14 @@ import { marked } from 'marked'
 
 import { registerHighlightContributions, renderMarkdown as renderMarkdownCommon } from '@sourcegraph/common'
 
+/**
+ * Supported URIs to render as links in outputted markdown.
+ * - https?: Web
+ * - vscode: VS Code URL scheme (open in editor)
+ * - command: VS Code command scheme (e.g. command:workbench.action.goToLine)
+ */
+const ALLOWED_URI_REGEXP = /^((https?|vscode):\/\/[^\s#$./?].\S*|command:.*)$/i
```
Comment (toolmantim) [inline client/cody-shared/src/chat/markdown.ts:11]: Do we care that this allows arbitrary commands? Or should we allow only the welcome guide?

**Review by toolmantim — APPROVED** [review]
Woohoo! Added one note about the regexp, and if we should allow only this specific command.

File: client/cody-shared/src/chat/markdown.ts (hunk 1 above)
Comment (umpox) [reply client/cody-shared/src/chat/markdown.ts:11]: That's a good point actually, I don't see much value in allowing arbitrary ones (and lots of downside). I'll update and double check everything

Comment (umpox) [reply client/cody-shared/src/chat/markdown.ts]: Updated! Confirmed all links still work and arbitrary command do not.

**Review by philipp-spiess — APPROVED** [review]

**Pull request merged**